### PR TITLE
Remove references of deprecated Shift IL Opcode

### PR DESCRIPTION
**Pull request by wbh123456**
Remove all references to the IL Opcodes in the `Shift` category listed in #2657.

Issue: #2657
Signed-off-by: Bohao(Aaron) Wang <aaronwang0407@gmail.com>

### CODE REVIEW

File: compiler/il/OMRILOps.hpp [hunk 1]
```diff
@@ -866,13 +866,12 @@ class ILOpCode
       return TR::BadILOp;
       }
 
+//TO DO: remove this function as unsigned shfit opcodes have been deprecated
```
Comment (Leonardo2718) [inline compiler/il/OMRILOps.hpp]: As the comment suggests, I think this function can just be removed.

Comment (wbh123456) [reply compiler/il/OMRILOps.hpp]: May I remove the function in another PR as this is outside of scope of this PR. I will remove the function along with all other similar functions as listed in https://github.com/eclipse/omr/issues/2657#issuecomment-508139949.

Comment (fjeremic) [reply compiler/il/OMRILOps.hpp]: Sounds like you may have wanted to write in the code then:

```
// TODO (#2657): ...
```

Please try to use the canonical name "TODO" as is the convention in the rest of the codebase. For example someone searching for todo's to fix is unlikely to search for "to do" (with a space) as such convention is not used anywhere in the codebase currently.

Comment (wbh123456) [reply compiler/il/OMRILOps.hpp]: Fixed in the new force-push.

**Comment by fjeremic** [comment]
I'll delegate to @Leonardo2718 to assign himself, approve, and merge.

@genie-omr build all

**Comment by Leonardo2718** [comment]
@wbh123456 can look into the linux_390-64 failure?

I'm restarting the linux_x86-64_cmprssptrs build as it failed due to infra issues.

@genie-omr build xcrlinux

File: compiler/z/codegen/S390Peephole.cpp [hunk 2]
```diff
@@ -2046,7 +2046,7 @@ TR_S390PostRAPeephole::revertTo32BitShift()
    // Note the NOT in front of second boolean expr. pair
    TR::InstOpCode::Mnemonic oldOpCode = instr->getOpCodeValue();
    if ((oldOpCode == TR::InstOpCode::SLLG || oldOpCode == TR::InstOpCode::SLAG)
-         && !(instr->getNode()->getOpCodeValue() == TR::ishl || instr->getNode()->getOpCodeValue() == TR::iushl))
+         && !instr->getNode()->getOpCodeValue() == TR::ishl)
```
Comment (fjeremic) [inline compiler/z/codegen/S390Peephole.cpp]: This looks like a bug. I think you meant:

```
&& instr->getNode()->getOpCodeValue() != TR::ishl
```

I suspect this is the reason we have PR build failures on Linux on Z.

Comment (fjeremic) [reply compiler/z/codegen/S390Peephole.cpp]: Hooray for Tril unit testing by the way. Not long ago this would have slid by unnoticed!

Comment (wbh123456) [reply compiler/z/codegen/S390Peephole.cpp]: Right, I should have done `&& !(instr->getNode()->getOpCodeValue() == TR::ishl)` or just `&& instr->getNode()->getOpCodeValue() != TR::ishl`. Fixed that in the new force push.

**Comment by Leonardo2718** [comment]
@genie-omr build all

**Comment by wbh123456** [comment]
@Leonardo2718 This PR should be ready for merge.

**Comment by Leonardo2718** [comment]
@genie-omr build all

**Pull request merged**